pkgs/development/libraries/xapian/default.nix

```diff
@@ -18,6 +18,13 @@ let
 
     doCheck = true;
 
+    # the configure script thinks that Darwin has ___exp10
+    # but it’s not available on my systems (or hydra apparently)
+    postConfigure = stdenv.lib.optionalString stdenv.isDarwin ''
+      substituteInPlace config.h \
+        --replace "#define HAVE___EXP10 1" "#undef HAVE___EXP10"
```

Comment:
"exp10" is a glibc extension, but *BSD usually provide "__exp10":

http://www.manpagez.com/man/3/__exp10/

I don't know why it's broken on macOS now, but this seems to fix it.